frontend/src/services.js/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+
+const baseUrl = "http://localhost:8000/api";
```

Comment:
Can we make this an environment variable so that we can swap this out when we run our app in production?

Reply:
Made this change!